thermo_nerf/nerfacto_config/config_nerfacto.py

```diff
@@ -30,7 +30,7 @@
                 ),
             ),
         ),
-        model=NerfactoModelConfig(eval_num_rays_per_chunk=1 << 15),
+        model=ThermalNerfactoModel_Config(eval_num_rays_per_chunk=1 << 15),
```

Comment:
Sorry the class name was good. This new name doesn't follow our standard. It's the variable that must be renamed `thermalnerfacto_config=ThermalNerfactoModelConfig()

Same for the other one